common/p_map.cpp

```diff
@@ -2774,7 +2775,13 @@ static BOOL PIT_DoomRadiusAttack(AActor* thing)
 		dist = 0;
 
 	if (dist >= bombdamage)
-		return true;	// out of range
+	{
+		if (bombsource && bombsource->player)
```

Comment:
If a condition block is spread across multiple lines, could you please add braces to them?